### ISSUE TITLE
First Issue -- Use this for the forking demo!!

### ISSUE THREAD

**Issue by shiinasugioka** [swecc-uw/swecc-website#63]
1. Assign yourself to this issue (click **Assignees** and then select yourself)
<img width="329" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/5084358e-053f-4a8f-850a-77b5e960ebb1">
Assign yourself here.
<img width="340" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/2ebb0923-1be1-45dd-abf4-a8441d3c034b">

2. Fork the repository
3. Clone your fork locally
4. Create a new branch with the number of the issue you're tackling (this issue's number)
5. Inside your new branch, add your name to a new line of `contributors.txt`
<img width="898" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/6b5ca53e-ce85-45a1-b7af-e365000695ba">

6. Commit and push your changes
7. Navigate back to the SWECC repo on GitHub, and go to the `Pull Requests` tab at the top. 
<img width="446" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/b2902f68-5a4b-4263-a724-914019bb1783">

8. Click the green `New Pull Request` button to open a new PR.
9. Select the blue link to `compare across forks`.
<img width="809" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/f6420ebe-44ef-41fb-a5ad-708e00ce1755">

10. Leave the left drop down as is, and change the right drop down to the branch in your forked repo that contains the code you want to merge. Make sure this says `<your-github-username>/swecc-website`
<img width="899" alt="image" src="https://github.com/swecc-uw/swecc-website/assets/91639423/afb3670b-1495-4a2d-b98b-75da77886330">

11. Fill in the PR, and assign Shiina (shiinasugioka) as a reviewer
12. Label the PR `SWECC Labs: New Contributor` and submit
13. Although you normally don't want to do this, please remove yourself from `Assignees` under this issue. We had you assign yourself as practice since that's what normally should happen but GitHub limits the number of assignees to 10 people so just for this practice issue, just take your name off once you've made your pull request : )
14. Confirm that your PR has no code reviews and comments. Your code will be merged by an officer once approved!
15. You're done! Congrats, that was your first official commit in SWECC Labs

**Comment by shiinasugioka** [comment]
This is where you want to make comments updating your work status!

For example:
"Currently in progress. Blocked by this issue, XXX, but doing more research into how I can fix this."
"Just assigned myself to the issue."
"Almost done with the code. Will make a PR once I delete comments and clean my code."

Feel free to leave comments here as practice. You can also tag people (@shiinasugioka ) and add screenshots if you have specific questions!
